stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -597,13 +595,13 @@ LogicalResult VhloBytecodeInterface::writeAttribute(
             write(attr, writer);
             return success();
           })
-      .Case<ArrayV1Attr, DenseIntOrFPElementsV1Attr, DictionaryV1Attr,
-            FlatSymbolRefV1Attr, FloatV1Attr, IntegerV1Attr, StringV1Attr,
-            TypeV1Attr>([&](auto attr) {
-        LOG_WRITE_CALL;  // Forked attrs
-        write(attr, writer);
-        return success();
-      })
+      .Case<ArrayV1Attr, DictionaryV1Attr, FlatSymbolRefV1Attr, FloatV1Attr,
```

Comment:
This split for `.Case` above and `.Case` here is that some are upstream and some are StableHLO. We can merge these two case statements since this distinction no longer matters (maybe this was done in a different PR)

Reply:
Yeah, same here :)